include/certifier_framework.h

```diff
@@ -234,11 +234,20 @@ class cc_trust_manager {
 
   // Each of the enclave types have bespoke initialization
 
+  // Interface ignored through SWIG bindings
   bool initialize_simulated_enclave(
       const string &serialized_attest_key,
       const string &measurement,
       const string &serialized_attest_endorsement);
 
+  // Interface invoked through Python apps
+  bool initialize_simulated_enclave(const byte *serialized_attest_key,
```

Comment:
@jlmucb said: If you can remove the overloading by renaming this as `python_initialize...`, that is perfectly fine too.

We can then get rid of the hacky shell script Perl replacement logic.